CHANGES.md

```diff
@@ -1,3 +1,43 @@
+## edge-19-2.4
+
+* CLI
+  * Implemented `proxy-cpu-limit` and `proxy-memory-limit` for setting the
+    proxy resources limits (`proxy-cpu` and `proxy-memory` were deprecated in
+    favor of `proxy-cpu-request` and `proxy-memory-request`) (thanks @TwinProduction!)
+  * Updated the `inject` and `uninject` subcommands to issue warnings when
+    resources lack a `Kind` property (thanks @Pothulapati!)
+  * Unhid the `install-cni` command and its flags, and tweaked their descriptions
+  * Fixed handling of kubeconfig server urls that include paths
+  * Added the `doc` command for generating docs for the CLI commands
+  * Removed the "experimental" flag from auto-inject
+  * Updated the `profile` help text to match the other commands
+  * Added the flags `-update` and `-pretty-diff` to tests to allow overwriting
+    fixtures and to print the full text of the fixtures upon mismatches
```

Comment:
Even though these flags apply to the CLI tests, I'm worried that folks will confuse them for actual CLI flags if we include them in this section. It looks like in previous releases we've included a top-level bullet point called "Internals". Mind adding that as a separate section at the bottom of the changelog? Something like:

```diff
diff --git a/CHANGES.md b/CHANGES.md
index 3ed1347c..fdcaa91d 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -11,8 +11,6 @@
   * Added the `doc` command for generating docs for the CLI commands
   * Removed the "experimental" flag from auto-inject
   * Updated the `profile` help text to match the other commands
-  * Added the flags `-update` and `-pretty-diff` to tests to allow overwriting
-    fixtures and to print the full text of the fixtures upon mismatches
   * Added the "ep" alias for the `endpoints` command (also @Pothulapati!)
   * Split the install templates by component
   * Added a validator for the `proxy-log-level` flag
@@ -37,6 +35,9 @@
     if there is only TCP traffic then the dropdowns don't end up empty
   * Ensured that all the tooltips in Grafana displaying the series are shared
     across all the graphs
+* Internals
+  * Added the flags `-update` and `-pretty-diff` to tests to allow overwriting
+    fixtures and to print the full text of the fixtures upon mismatches
 
 ## stable-2.2.0
 
```